<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-6EUUVVQ8" failures="1" tests="3" name="Test" time="11.915" errors="0" timestamp="2025-01-03T20:22:25 IST">
  <testcase classname="session46_ListenerClass.TestClass" name="testlogo" time="3.055"/>
  <testcase classname="session46_ListenerClass.TestClass" name="testUrl" time="0.011">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at session46_ListenerClass.TestClass.testUrl(TestClass.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testUrl -->
  <testcase classname="session46_ListenerClass.TestClass" name="testTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testTitle -->
</testsuite> <!-- Test -->
